src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/AbstractIntegrationTest.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.VisualStudio.Razor.IntegrationTests;
 /// <item><description><see cref="IDisposable.Dispose"/></description></item>
 /// </list>
 /// </remarks>
-[IdeSettings(MinVersion = VisualStudioVersion.VS2022, RootSuffix = "RoslynDev", MaxAttempts = 2)]
+[IdeSettings(MinVersion = VisualStudioVersion.VS2022, RootSuffix = "RoslynDev", MaxAttempts = 1)]
```

Comment:
I'm convinced this was just covering up for flaky behavior (like ordering in FAR tests) and allowing it to be missed during initial development.

Reply:
I wouldn't say you're wrong, but there absolutely is some flakiness in the whole test system, that this could expose too. eg, when the experimental instance starts and errors because it couldn't load packages etc. Maybe they don't happen in CI though? 🤷‍♂️

Reply:
I have seen the package load issues. For whatever reason I don't think I've ever seen them on the CI. I was discussing with @allisonchou last week and we think that it's possible for MEF cache's for a non-hive instance to pollute hive instances (because we've both run into scenarios where we're unable to get a clean hive [even by deleting the entire hive and MEF cache for VS] until we uninstall and re-install VS). If we've guessed at that correctly then CI instances would be safe because they get re-created clean each time.

More generally though I agree the current instability is too great to turn off the retry, even if I'm convinced it's what got us here in the first place. Part of what I'm doing in this PR is turning knobs to see if we can get generalized stability up.

Reply:
Good theory. If they never happen in CI, then i'm fine with removing retries, as even when it does happen locally, I find the retry system more annoying than not, and would be happy to just retry them myself, manually.